components/brave_wallet/browser/filecoin_keyring.cc

```diff
@@ -137,7 +138,9 @@ std::string FilecoinKeyring::GetAddressInternal(HDKeyBase* hd_key_base) const {
   // TODO(spylogsster): Get network from settings.
   return FilAddress::FromUncompressedPublicKey(
              hd_key->GetUncompressedPublicKey(),
-             mojom::FilecoinAddressProtocol::SECP256K1, mojom::kFilecoinTestnet)
+             mojom::FilecoinAddressProtocol::SECP256K1,
+             IsFilecoinTestnetEnabled() ? mojom::kFilecoinTestnet
+                                        : mojom::kFilecoinMainnet)
```

Comment:
As mentioned in DM, please add comment here to explain why this is fine for now.